src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -71,18 +74,22 @@ The following table shows the differences between both solutions:
   </tbody>
 </table>
 
-## Requirements
+For more on migrating from the previous integrations to this integration, see [Migration](#migrate).
+
+## Requirements [#requirements]
 
-The Azure integration activation process requires you to:
+To use the Azure Monitor integration, you must:
```

Comment:
Are these requirements or steps to getting this set up? Maybe it's worth reframing it that way?